dev-util/byacc/byacc-20180510.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="Berkeley implementation of Yacc"
+DESCRIPTION="Berkeley Yacc (byacc) is generally conceded to be the best yacc \
+variant available. In contrast to bison, it is written to avoid dependencies \
+upon a particular compiler."
+HOMEPAGE="http://invisible-island.net/byacc/byacc.html"
+COPYRIGHT="2002-2013,2018 by Thomas E. Dickey"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="ftp://ftp.invisible-island.net/byacc/byacc-$portVersion.tgz"
```

Comment:
Here's a variant (which works fine):
```diff
-SOURCE_URI="ftp://ftp.invisible-island.net/byacc/byacc-$portVersion.tgz"
+SOURCE_URI="https://invisible-mirror.net/archives/byacc/byacc-$portVersion.tgz"
```
That said, the current `SOURCE_URI` is also OK, of course.

Reply:
Yeah, also used that one, but I thought I'd best use the one from the previous recipe ;)